charts/opensearch/values.yaml

```diff
@@ -205,6 +205,8 @@ persistence:
   labels:
     # Add default labels for the volumeClaimTemplate of the StatefulSet
     enabled: false
+    # Add custom labels for the volumeClaimTemplate of the StatefulSet
+    additionalLabels: {}
   # OpenSearch Persistent Volume Storage Class
```

Comment:
Thanks @simonstumpf there is already `Values.persistence.labels` which is enabled to false by default, what is the expected behaviour here for `additionalLabels` do we have to `Values.persistence.labels.enabled` to true? From code looks to me like we dont have to enable and can still pass the `additionalLabels` to the PVC's.

Reply:
Hi @simonstumpf would you mind check the comments from @prudhvigodithi ?

Thanks.

Reply:
Hi all, 
Sorry I was on a mini vacation, just returned to the office. 

@prudhvigodithi : Sorry I totally ignored this labels.enabled field during my first commit. 
In my point of view the logic should be that regardless of labels.enabled, the additional labels should always be applied. 

I therefore did some changes to the labels logic, could you have another look?

Reply:
Thanks @simonstumpf, so regardless of the `labels.enabled` (enabled or disabled) if the `persistence.labels.additionalLabels` is passed then the `additionalLabels` will be applied to the PVC's right ? Also again regardless of the `labels.enabled` (enabled or disabled) if the `persistence.labels.additionalLabels` is not passed (no labels) then there wont be any `additionalLabels` applied, am I right here?

Reply:
@prudhvigodithi  Yes, you are right